frontend/amundsen_application/static/js/components/Lineage/Graph/index.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+import * as ReactDOMServer from 'react-dom/server';
+import { Lineage } from 'interfaces';
+import { LeftArrowIcon, RightArrowIcon } from '../../SVGIcons';
+import lineageWrapper from './wrapper';
+import { LineageChart } from './chart';
+import { CHART_DEFAULT_DIMENSIONS, LINEAGE_SCENE_MARGIN } from './constants';
+import './styles.scss';
+
+export interface GraphProps {
+  lineage: Lineage;
+}
+
+export const Graph: React.FC<GraphProps> = ({ lineage }: GraphProps) => {
+  const rootRef = React.useRef<HTMLDivElement>(null);
+  const [scene, setScene] = React.useState<LineageChart | null>(null);
+
+  // Setting up margins & render screen for the graph.
+  React.useEffect(() => {
+    if (!scene && rootRef.current) {
+      const containerDimensions = rootRef.current.getBoundingClientRect();
+      const dimensions = {
+        width:
+          (containerDimensions.width || CHART_DEFAULT_DIMENSIONS.width) -
+          (LINEAGE_SCENE_MARGIN.left + LINEAGE_SCENE_MARGIN.right),
+        height:
+          (containerDimensions.height || CHART_DEFAULT_DIMENSIONS.height) -
+          (LINEAGE_SCENE_MARGIN.top + LINEAGE_SCENE_MARGIN.bottom),
+      };
+
+      const currentScene = lineageWrapper.create(
+        rootRef.current,
+        lineage,
+        dimensions,
+        {
+          upstream: `${ReactDOMServer.renderToString(
+            <div className="text-title-w2">
+              <LeftArrowIcon fill="#acacc0" />
+              <span>Upstream</span>
```

Comment:
We are trying to get our strings out of the components render function, in this case I would create a constant in this file named UPSTREAM_ARROW_LABEL